Compilation error when using USD 20.11

**Describe the bug**
The most recent version of arnold-usd doesn't compile with 20.11, it errors with:
`libs\render_delegate\reader.cpp(15): fatal error C1083: Cannot open include file: 'pxr/imaging/hio/image.h': No such file or directory`

--- a/libs/render_delegate/reader.cpp
+++ b/libs/render_delegate/reader.cpp
@@ -12,7 +12,6 @@
 #include "pxr/imaging/hd/renderPass.h"
 #include "pxr/imaging/hd/rendererPlugin.h"
 #include "pxr/imaging/hd/rendererPluginRegistry.h"
-#include "pxr/imaging/hio/image.h"
 #include "pxr/usd/sdf/path.h"
 #include "pxr/usd/usd/stage.h"
 #include "pxr/usd/usdGeom/camera.h"

